src/templates/pycontw-2025/contents/zh/about/code-of-conduct.html

```diff
@@ -101,7 +101,11 @@ <h2>授權 (CC BY-SA 3.0 TW)</h2>
     授權條款。</p>
 <img src="https://i.imgur.com/cVjN760.jpg =200x"></img>
 
-<p>PyCon TW 2024 行為準則從 2024 年 02 月 20 日起開始生效，PyCon Taiwan 保留最終解釋與調整權，如有需要我們將隨時修改此份準則，並將儘速更新與公告予您。</p>
+<p>PyCon TW 2025 行為準則從 2025 年 02 月 20 日起開始生效，PyCon Taiwan 保留最終解釋與調整權，如有需要我們將隨時修改此份準則，並將儘速更新與公告予您。</p>
 <p>感謝。</p>
 
+<p>
+    當您的投稿被接受並成為 PyConTW 大會講者後，請注意，我們並不會主動提供交通或大會門票等財務補助。如需申請相關補助，請參閱我們的財務<a href="https://tw.pycon.org/2025/zh-hant/registration/financial-aid">補助說明</a>，了解如何提交申請。若有其他問題，也可以透過電子郵件聯絡我們：organizers@python.tw。
```

Comment:
please check how these variables are used in other template
```suggestion
    當您的投稿被接受並成為 PyConTW 大會講者後，請注意，我們並不會主動提供交通或大會門票等財務補助。如需申請相關補助，請參閱我們的財務<a href="{{ FRONTEND_HOST }}/{{ LANGUAGE_CODE }}/registration/financial-aid">補助說明</a>，了解如何提交申請。若有其他問題，也可以透過電子郵件聯絡我們：organizers@python.tw。
```